library/src/main/java/com/datatorrent/lib/db/jdbc/AbstractJdbcPollInputOperator.java

```diff
@@ -717,6 +729,24 @@ public void setKey(String key)
     this.key = key;
   }
 
+  /**
+   * gets the Result Limit size, parameter to limit the number of results
+   * to fetch in one query by the Poller partition.
+   */
+  public int getResultLimit()
+  {
+    return resultLimit;
```

Comment:
querySizeLimit is not sounding correct to me because we cannot have a limit on the size of the query we can only limit the result set.